aspnetcore/tutorials/signalr.md

```diff
@@ -1,23 +1,27 @@
 ---
-title: "Tutorial: Get started with SignalR on ASP.NET Core"
+title: "Get started with SignalR on ASP.NET Core"
 author: tdykstra
 description: In this tutorial, you create a chat app that uses SignalR for ASP.NET Core.
 monikerRange: '>= aspnetcore-2.1'
 ms.author: tdykstra
 ms.custom: mvc
+ms.topic: tutorial
```

Comment:
Remove ms.topic. This will automatically be applied. See the glob pattern in *docfx.json*.